### PR TITLE
ZEIT -> Vercel

### PR DESCRIPTION
**Pull request by Qix-**
@rauchg @leo can you take a look and let me know if this is okay for Vercel?

The package needs to be migrated on npm, too, unless you want to approach this some other way.

Also, would you be willing to allow me publishing privs again? 😅

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 ZEIT, Inc.
+Copyright (c) 2017-2020 ZEIT, Inc.
+Copyright (c) 2020-2021 Vercel, Inc.
```
Comment (leerob) [inline LICENSE]:
```suggestion
Copyright (c) 2021 Vercel, Inc.
```

File: README.md [hunk 2]
```diff
@@ -63,4 +63,7 @@ will be the only hook that is executed.
 > as environment variables prior to installing.
 
 # License
-Copyright &copy; 2018 by ZEIT, Inc. Released under the [MIT License](LICENSE).
+Copyright &copy; 2017-2020 by ZEIT, Inc.
+Copyright &copy; 2020-2021 by Vercel, Inc.
+
```
Comment (leerob) [inline README.md]:
```suggestion
Copyright &copy; 2021 by Vercel, Inc.

```

File: package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@zeit/git-hooks",
+  "name": "@vercel/git-hooks",
   "version": "0.2.1",
   "description": "No nonsense Git hook management",
-  "repository": "zeit/git-hooks",
-  "author": "Josh Junon <junon@zeit.co>",
+  "repository": "vercel/git-hooks",
+  "author": "Josh Junon <josh@junon.me>",
```
Comment (leerob) [inline package.json:6]: Edit: removed!

File: README.md (hunk 2 above)
Comment (Qix-) [reply README.md]: This isn't legally accurate, though. This library pre-dates 2021.

File: package.json (hunk 3 above)
Comment (Qix-) [reply package.json:6]: Could you explain this change, please? Do I not maintain this anymore?

**Comment by Qix-** [comment]
@leo @rauchg I don't want to burn any bridges here but perhaps Vercel would like to simply assume full ownership of this package? It seems as though I'm not being equipped with enough permissions to continue doing so. I don't want to miss any 'dropped hints'.

File: package.json (hunk 3 above)
Comment (leerob) [reply package.json:6]: Sorry, I will be honest I didn't know you created this entirely. I saw an old ZEIT email so I figured it needed to be removed. Happy to keep you as the author!

File: README.md (hunk 2 above)
Comment (leerob) [reply README.md]: It's the standard for the other repos in our GitHub org 👍

**Comment by leerob** [comment]
As mentioned in https://github.com/vercel/git-hooks/pull/7#issuecomment-815965642, happy to add you as a maintainer.

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Thank you!

**Pull request merged**